express-rest-api/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "express-rest-api-example",
+  "version": "1.0.0",
+  "description": "Rest API example with Express and Supertest",
+  "scripts": {
+    "test": "mocha",
+    "start": "node ./server/index.js"
+  },
+  "directories": {
+    "test": "./test"
+  },
+  "devDependencies": {
+    "mocha": "^6.0.2",
```

Comment:
Please use latest mocha`^6.2.1` like [this](https://github.com/mochajs/mocha-examples/commit/572a5750dbc760729ebfa96b6b754bc793eaa587).